src/com/epam/javacourse/homework2/HomeWork.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.javacourse.homework2;
+
+import java.util.Arrays;
+
+public class HomeWork {
+
+    public static void main(String[] args) {
+        // Fibonacci
+        Fibonacci fibonacci = new Fibonacci();
+        fibonacci.getFibonacciRow(10);
+
+        //Palindrome
+        Palindrome palindrome = new Palindrome();
+        System.out.println("\nIs 121 Palindrome: " + palindrome.isPalindrome(121));
+
+        //Sqrt
+        Sqrt sqrt = new Sqrt();
+        final int mySqrt = sqrt.mySqrt(4);
+        System.out.println(mySqrt);
+
+        //TwoSums
+        TwoSums twosums = new TwoSums();
+        final int[] twoSum = twosums.twoSum(new int[]{2, 7, 11, 15}, 9);
```

Comment:
If you still want to play with `final` after getting rid of variables, you can put it into the instance creation line like
`final TwoSums twosums = new TwoSums();`